docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3'
+
+services:
+  db:
+    image: mysql:latest
```

Comment:
Is this intentional? This will grab MySQL 8.0. Is that the version used in stage/prod?